CODE_OF_CONDUCT.md

```diff
@@ -1,76 +1,120 @@
+The Ombudsperson (currently Geoffrey Lovelace, email ombudsperson@black-holes.org) offers confidential, neutral, and informal conflict resolution services to members of the SXS Collaboration. All collaboration members should feel free to contact the Ombudsperson to informally and confidentially discuss any concerns that they might have regarding conflicts, problems, violations of the code of conduct, or any other concerns they might have.
 
-## Scope
+After speaking with Ombudsperson, if you wish, you can choose to ask the Ombudsperson to bring your concerns to the attention of the SXS Executive Committee or to other people at the appropriate institutions, and you can also choose to report your concerns formally to the SXS Executive Committee (as outlined below). Otherwise, unless there is a serious issue of safety, your conversations with the Ombudsperson will remain confidential.
 
-This Code of Conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community. Examples of
-representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
-representative at an online or offline event. Representation of a project may be
-further defined and clarified by project maintainers.
+We have adopted the same policy as the LIGO Scientific Collaboration Ombudsperson; see this document, replacing “LSC” with “SXS Collaboration”, and “LSC Spokesperson” with “the SXS Executive Committee”, for more details about the Ombudsperson’s role see the [LIGO document](https://dcc.ligo.org/public/0099/M1300006/001/LSCOmbudsperson.pdf).
 
-## Enforcement
+###  Student/Postdoc Advocate
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at support@github.com. All
-complaints will be reviewed and investigated and will result in a response that
-is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
+The Student/Postdoc Advocate (currently Masha Okounkova, email sxs-advocate@black- holes.org) is someone who will advocate positively for the people in the collaboration with less power, specifically high school students, undergraduate students, summer REU students, grad students, and postdocs. It may sometimes be daunting to go to professors, so please feel free to talk to the Advocate about any issues and concerns you may have in the collaboration, especially concerns about culture and environment. The Advocate can bring up (anonymously) any of your concerns with the SXS executive committee if you wish. Other- wise, unless theres a serious issue of safety, the Advocate will keep the Ombudsperson in the loop but otherwise keep your conversation confidential.
```

Comment:
The email address on this line is broken, so does not get rendered as a clickable link

Reply:
Also, the auto-hyphenation of "Otherwise" has made it into the markdown and should be un-hyphenated here.